recipes/cassandra/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cassandra" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "b35a2b485470773e24fa928c928bdbe8a53a469dd4e441049f3bab6c114f0612" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://raw.githubusercontent.com/MaginnGroup/Cassandra/master/releases/Cassandra_V{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py3k]
```

Comment:
python 2.7 has reached end of life. do you really want to exclude python 3?

Reply:
Migration to Python3.x is in the works.